src/wait-for.js

```diff
@@ -89,6 +90,12 @@ function waitFor(
         await new Promise(r => setImmediate(r))
       }
     } else {
+      try {
```

Comment:
This change was needed to keep these warnings intact. Without it, `getWindowFromNode(container)` throws and the error messages do not match the expected error. I thought it best to keep the warnings/behavior the same rather than update the test snapshots to match the new error message.

 https://github.com/testing-library/dom-testing-library/blob/master/src/__tests__/base-queries-warn-on-invalid-container.js#L95-L127